app/assets/stylesheets/application.scss

```diff
@@ -13,3 +13,7 @@
  */
 @import "bootstrap-sprockets";
 @import "bootstrap";
+
+.discount-price {
+  color: #ff0000;
```

Comment:
Color literals like `#ff0000` should only be used in variable declarations; they should be referred to via variable everywhere else.